LICENSE.txt

```diff
@@ -9,7 +9,7 @@ software.
 
 1. Definitions
 
-The terms "reproduce," "reproduction," "derivative works," and "distribution"
+The terms "reproduce", "reproduction", "derivative works" and "distribution"
```

Comment:
Interesting...  I wonder if that is a legalese thing?  Because the same typo exists in the original MsPL license:

https://opensource.org/licenses/MS-PL